presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadataFactory.java

```diff
@@ -38,6 +38,7 @@
     private final boolean bucketWritingEnabled;
     private final boolean skipDeletionForAlter;
     private final boolean writesToNonManagedTablesEnabled;
+    private final boolean createNonManagedTableEnabled;
```

Comment:
Modified. I would rather go with `createNonManagedTableEnabled` to make it consistent with the property name `hive.create-non-managed-table-enabled`.